src/data/code-samples/en/accessibility-hint/jetpack-compose.md

```diff
@@ -0,0 +1,12 @@
+# Accessibility hint - Jetpack Compose
+
+In Jetpack Compose, you can use [`contentDescription`](https://developer.android.com/reference/kotlin/androidx/compose/ui/semantics/package-summary#(androidx.compose.ui.semantics.SemanticsPropertyReceiver).contentDescription()) parameter inside [`semantics`](https://developer.android.com/reference/kotlin/androidx/compose/ui/semantics/package-summary#(androidx.compose.ui.Modifier).semantics(kotlin.Boolean,kotlin.Function1)) modifier block to set a hint.
```

Comment:
Also add documentation + code for `placeholder` (that's the equivalent for EditText hint)